PubNub/Core/PubNub+Core.m

```diff
@@ -457,7 +457,7 @@ - (void)processOperation:(PNOperationType)operationType withParameters:(PNReques
 
 - (void)cancelAllLongPollingOperations {
     
-    [self.subscriptionNetwork cancelAllRequests];
+    [self.subscriptionNetwork cancelAllOperationsWithURLPrefix:@"/v2/subscribe/"];
```

Comment:
@jzucker2 in new commit I've changed all this. String has been moved to constant in appropriate place.